README.md

```diff
@@ -73,6 +73,11 @@ For example,
 - Go - https://github.com/grpc/grpc-go/blob/96cefb43cfc8b2cd3fed9f19f59830bc69e30093/credentials/oauth/oauth.go#L60
 - Python - https://github.com/google/oauth2client/blob/0d1c814779c21503307b2f255dabcf24b2a107ac/oauth2client/service_account.py#L552
 
+## Other languages
```

Comment:
"Other" sounds wrong because the word makes sense only when there is a main subject of discussion.

Reply:
Yeah just `Examples` is fine